rust/xaynet-client/src/state_machine/phases/phase.rs

```diff
@@ -0,0 +1,212 @@
+use derive_more::From;
+use thiserror::Error;
+
+use xaynet_core::{
+    common::RoundParameters,
+    crypto::SigningKeyPair,
+    mask::MaskConfig,
+    message::Payload,
+};
+
+use super::{Awaiting, NewRound, Sum, Sum2, Update};
+use crate::{
+    settings::{MaxMessageSize, Settings},
+    state_machine::{io::StateMachineIO, StateMachine, TransitionOutcome},
+    MessageEncoder,
+};
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct State<P> {
+    /// data specific to the current phase
+    pub phase: P,
```

Comment:
I would rename it maybe to something like `phase_state` / `individual` or `private`. 
`phase` is a bit misleading here because we also have a `Phase` struct that uses `State`

Reply:
Initially these were `phase_state` and `shared_state` but it was very verbose. I like `Private` since it's a natural counterpart to `Shared`. Thanks for the suggestion!